src/markdown-pages/docs/cli-command-reference/runs-get.md

```diff
@@ -17,8 +17,9 @@ Use the tables provided to view the options for filtering test results, and for
 | :---- | :-------- | 
 | `--name`  | Use the `--name` option to query the status of a particular test run.  |
 | `--requestor`| Use the `--requestor` option to view results of test runs that were submitted to the ecosystem by a specified user. The default is the current user name that is specified in the access token.|
```

Comment:
> Use the `--requestor` option to view results of test runs that were submitted to the ecosystem by a specified user. The default is the current user name that is specified in the access token.

That's wrong. Should be:

Use the `--requestor` option to filter the results of test runs, such that only those  submitted to the ecosystem by a specified user are shown. If not specified, the default behaviour is to show all the test results from any user.

Reply:
Thank you - have corrected in latest PR for next